server/content-security-policy.js

```diff
@@ -51,6 +51,8 @@ const COMMON_DIRECTIVES = {
     'https://www.google.com',
     'https://api.cryptonator.com',
     'https://plausible.io',
+    'opencollective-production.s3.us-west-1.amazonaws.com',
+    'opencollective-staging.s3.us-west-1.amazonaws.com',
```

Comment:
I suppose we can remove them since we download from `/download` (not client-side)?

```suggestion
```

Reply:
@Betree this is needed for the PDF Viewer that loads content from these buckets

Reply:
@Betree this is needed for the PDF Viewer that loads content from these buckets

Reply:
Makes sense, in this case we should move them out of `COMMON_DIRECTIVES`, if you look a bit below there are specific directives for `production` and `staging` that get merged with the main config:
 https://github.com/opencollective/opencollective-frontend/blob/9bce10537690ad304ec6986e7f46ea54c066429f/server/content-security-policy.js#L151

Also, we have entries for both `s3.us-west-1` and `s3-us-west-1` in the DB, unless we want to consolidate now we should support both formats (as done in the regex)